Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift

```diff
@@ -89,6 +90,15 @@ public class JSClosure: JSFunction {
         }
     }
 
+    public required convenience init(jsValue: JSValue) {
+        switch jsValue {
+        case let .function(fun):
+            self.init { fun(arguments: $0) }
+        default:
+            fatalError()
+        }
+    }
+
```

Comment:
```suggestion
    public override class func construct(from value: JSValue) -> Self? {
        guard case let .function(fun) = value else { return nil }
        return JSClosure { fun(arguments: $0) } as? Self
    }

```

Reply:
This is already implemented here:

https://github.com/j-f1/forked-JavaScriptKit/blob/479ad20b07f36a2db1cb5a410f12a7ae587d1b49/Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift#L57-L59

Should I remove this `init(jsValue:)` declaration then?

Reply:
Yes, it would be more consistent interface.